lib/dockly/docker.rb

```diff
@@ -33,12 +33,27 @@ def build_env(hash = nil)
   end
 
   def generate!
-    image = generate_build
+    image = find_image_by_repotag || generate_build
     export_image(image)
   ensure
     cleanup([image]) if cleanup_images
   end
 
+  def export_only
+    if image = find_image_by_repotag
+      info "Found image by repo:tag: #{repo}:#{tag} - #{image.inspect}"
+      export_image(image)
+    else
+      raise "Could not find image"
+    end
+  end
+
+  def find_image_by_repotag
```

Comment:
Perhaps we would want to add a flag that will always rebuild, incase the last image was bad for some reason.